resources/views/components/toolbar/toolbar.blade.php

```diff
@@ -12,15 +12,20 @@
     @include('laravel-views::components.toolbar.search')
   </div>
 
-  {{-- Actions on the left --}}
-  <div class="flex gap-2 flex-1 justify-end items-center mb-4">
+  {{-- Actions on the right --}}
+  <div class="flex gap-2 flex-1 justify-end items-center mb-4 relative">
     {{-- Sorting --}}
     @if (isset($sortableBy) && $sortableBy->isNotEmpty())
       <div>
         @include('laravel-views::components.toolbar.sorting')
       </div>
     @endif
 
+    {{-- Loading Spinner --}}
```

Comment:
I like this idea, I've been thinking about where to  put this loading indicator, now that there is a toolbar component I think this is the place